Workflows/01_config_snakemake/config_general_MZB.yml

```diff
@@ -6,38 +6,53 @@ dir_models:  02_Models              # Folder where the base models will be saved
 dir_runs:    03_Runs                # Folder where model runs will be saved (with forcing)
 runname_ids: 
     Idai:                                                   # name of the model run
+        # Settings of the workflow
         tc_name: Idai                                       # name of the storm. For tropical cyclones, this can be the name of the TC
         region: sofala                                      # name of the region will be used in naming folders 
-        bbox_sfincs: "[34.33,-20.12,34.95,-19.30]"          # Bounding box for the flood model in format: [lon_min, lat_min, lon_max, lat_max]
-        bbox_dfm: "[32.3,42.5,-27.4,-9.5]"                  # Bounding box for the coastal hydrodynamic model in format: [lon_min, lon_max, lat_min, lat_max]
+        sid: 2019063S18038                                  # Storm id from IBTrACS
+            
+        # For all model setups
         start_time: 20190309 000000
-        end_time: 20190325 060000                   
-        forcing: era5_hourly                                # forcing dataset for the rainfall data (matching to the DataCatalog handle)
-        wind_forcing: spw_IBTrACS                           # wind and pressure dataset (used for coastal surge modelling)
+        end_time: 20190325 060000      
+        forcing: era5_hourly_zarr                    # forcing dataset for the rainfall data (matching to the DataCatalog handle)
+        wind_forcing: era5_hourly_zarr   # spw_IBTrACS                           # wind and pressure dataset (used for coastal surge modelling)
         bathy: 'gebco2024_MZB'                              # DataCatalog handle for the bathymetry dataset
+        ext_days: '9'                                       # the nr of days used to extend the spw file beyond the IBTrACS time period               
+
+        # Models selections
+        use_dfm: True                                       # use the coastal hydrodynamic model D-FM
+        use_wflow: True   
```

Comment:
I think right now 'use_wflow' and 'use_wave' are not pointing to really being used? Or does the user get an error if use_wflow is False? Maybe we could for now comment those lines and write in the description that this option has not been added yet